spoof-device.py

```diff
@@ -4,24 +4,34 @@
 
 devices = [evdev.InputDevice(path) for path in evdev.list_devices()]
 
+if not devices:
+    print('No device available')
```

Comment:
I think for clarity, this should be "No evdev devices found  on your system. Check your evdev installation and device permissions.".